competition/README.md

```diff
@@ -0,0 +1,16 @@
+# Competition
+
+This part is for all competitions.
+Codes of all competitions are stored by submoudule. If you need to obtain it, please initialize the submoudule after cloning PaddleScience.
+
+## submodule initialization
+
+    # for example IJCAI_2024_CAR
+    cd PaddleScience
+    git submodule update --init --recursive competition/IJCAI_2024_CAR
```

Comment:
````suggestion
``` sh
# for example IJCAI_2024_CAR
cd PaddleScience
git submodule update --init --recursive competition/IJCAI_2024_CAR
```
````